### DISCUSSION TITLE
Convert SVG to JPG.... i don't understand. ^^'

### DISCUSSION THREAD

**Discussion by JDragonB** [ImageMagick/ImageMagick#6828] [Help]
### ImageMagick version

7.1.1-21

### Operating system, version and so on

Windows 11 22H2 and PowerShell 5.1

### Description

Hello!

I don't understand the result after converting, see attach files.
When i look at the SVG with my browser, all is OK but after... 
Can you help me?

![Meteo](https://github.com/ImageMagick/ImageMagick/assets/68791091/dd28feb4-2813-4b7a-bfe9-936ec5b56e51)
![Meteo](https://github.com/ImageMagick/ImageMagick/assets/68791091/ae6770bc-b312-4cd6-9bb3-e1ea5ef14fd9)

**Comment by snibgo** [comment]
What do want changed? The background is white. If you want a different background such as Blue, insert "`-background Blue`" before reading the SVG.

**Reply by JDragonB** [reply]
Well... No ; they are a background picture ; the font is not the same and text with black color is... white.

**Comment by snibgo** [comment]
The font specified in the SVG is "Fantasy". If you don't have that font, software will use some other font.

The SVG contains no black text.

What delegate rasterized the SVG?

**Comment by JDragonB** [comment]
Fantasy is a standard font for SVG, all other software understant it...
